services/web/server/src/simcore_service_webserver/users/_handlers.py

```diff
@@ -42,6 +52,25 @@ async def wrapper(request: web.Request) -> web.StreamResponse:
 
         except UserNotFoundError as exc:
             raise web.HTTPNotFound(reason=f"{exc}") from exc
+        except MissingGroupExtraPropertiesForProductError as exc:
+            error_code = create_error_code(exc)
+            frontend_msg = FMSG_MISSING_CONFIG_WITH_OEC.format(error_code)
+            log_msg = create_troubleshotting_log_message(
+                message_to_user=frontend_msg,
+                error=exc,
+                error_code=error_code,
+                error_context=exc.error_context(),
+                tip="Row in `groups_extra_properties` for this product is missing.",
```

Comment:
So this field that you call `tip` is like a suggestion on how to solve the issue if this is seen in the logs?
If that's true I'd call it `action_to_take` or `recommended_action`